notebook-tests/tests/__init__.py

```diff
@@ -21,6 +21,8 @@
 
 def _find_notebook(notebook_file: str) -> str:
     path_to_notebook = os.path.join("colab", notebook_file)
+    if not os.path.exists(path_to_notebook):
```

Comment:
this was needed for local execution with pytest when you're in the same directory

Reply:
okay